packages/tools/devtools/devtools-core/api-report/devtools-core.alpha.api.md

```diff
@@ -16,6 +16,24 @@ export type ContainerKey = string;
 // @beta
 export function createDevtoolsLogger(baseLogger?: ITelemetryBaseLogger): IDevtoolsLogger;
 
+// @alpha
+export interface DataObjectProps {
```

Comment:
We should probably hold off on adding the DataObject APIs until we are more confident that they are what Pages actually wants. Sounds like there's still some ongoing discussion there.